test/system/practical_support_behaviors_test.rb

```diff
@@ -0,0 +1,122 @@
+require "application_system_test_case"
+
+class PracticalSupportBehaviorsTest < ApplicationSystemTestCase
+  before do
+    @user = create :user
+    @patient = create :patient, line: 'DC'
+  end
+
+  describe 'creating a new practical support entry' do
+    before { go_to_practical_support_tab }
+
+    it 'should start empty' do
+      within :css, '#existing-practical-supports' do
+        refute has_text? 'Category of support'
```

Comment:
has_text? only returns after the capybara default wait time, so using it when the expectation is negative adds unnecessary waits to test suite. Writing this as assert has_no_text? will still wait for page load but will avoid this issue